lanl/podman-quadlets/roles/auth/handlers/main.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: distrust ochami certificate
+  become: true
+  ansible.builtin.shell: 'trust anchor --store /tmp/ochami.pem; update-ca-trust'
```

Comment:
This *trusts* the certificate. To distrust, replace `--store` with `--remove`.

I went back and looked at the example Ansible I gave you and I see it's incorrect there as well, so I will fix that.

Reply:
Resolved with [18d0d6e](https://github.com/OpenCHAMI/deployment-recipes/pull/77/commits/18d0d6ecc63042294e2e443b1a86f6078a525abe).